sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.net8.0.cs

```diff
@@ -42,6 +43,10 @@ protected AvailabilitySetResource() { }
         public virtual bool HasData { get { throw null; } }
         public virtual Azure.Response<Azure.ResourceManager.Compute.AvailabilitySetResource> AddTag(string key, string value, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Compute.AvailabilitySetResource>> AddTagAsync(string key, string value, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response CancelMigrationToVirtualMachineScaleSet(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CancelMigrationToVirtualMachineScaleSetAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.ResourceManager.ArmOperation ConvertToVirtualMachineScaleSet(Azure.WaitUntil waitUntil, Azure.ResourceManager.Compute.Models.ConvertToVirtualMachineScaleSetContent content = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I think here, one could call `ConvertToVMSS` to start the process, and then they could call `CancelMigrationToVMSS` to cancel it, right?
If so, I think the names here do not align with each when the start operation says `Convert` but the cancel operation says "cancel migration`.
Is it possible that we could change one of them to be aligned with each other?

Reply:
there is a "StartMigrationToVirtualMachineScaleSet" operation to mirror CancelMigrationToVirtualMachieScaleSet.

ConvertToVmss is something else

Reply:
I see - then I might misunderstand these operations. Let us keep their current names.